solidity_deposit_contract/deposit_contract.sol

```diff
@@ -12,8 +12,8 @@
 pragma solidity 0.6.11;
 
 // This interface is designed to be compatible with the Vyper version.
-/// @notice This is the Ethereum 2.0 deposit contract interface.
-/// For more information see the Phase 0 specification under https://github.com/ethereum/eth2.0-specs
+/// @notice This is the Ethereum staking deposit contract interface.
+/// For more information see the Phase 0 specification under https://github.com/ethereum/consensus-specs
```

Comment:
I think we should leave it for the deposit contract so it can match the history accurately. 😄

Reply:
Yeah that makes sense! Will revert changes in that directory.